src/Compilers/Server/VBCSCompilerTests/VBCSCompiler.UnitTests.csproj

```diff
@@ -7,6 +7,7 @@
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <TargetFrameworks>$(RoslynPortableTargetFrameworks)</TargetFrameworks>
     <RuntimeIdentifiers>$(RoslynPortableRuntimeIdentifiers)</RuntimeIdentifiers>
+    <PlatformTarget>x86</PlatformTarget>
```

Comment:
You should set VBCSCompiler explciitly to AnyCPU not this to x86. 

I saw there's a place where it is set to x86 for bootstrap so you'll want to not override that case to AnyCPU